pyglossary/plugins/ebook_mobi.py

```diff
@@ -93,13 +87,32 @@
 		comment="Include index page",
 	),
 	"apply_css": StrOption(
-		disabled=True,
+		# disabled=True,
 		comment="Path to css file",
 	),
 	"cover_path": StrOption(
-		disabled=True,
+		# disabled=True,
 		comment="Path to cover file",
 	),
+	"group_size": IntOption(
+		comment="Roughly specify a max size of each xhtml file(in KB). "
+		"value less than 100 will round to 100.",
+	),
+	"hide_word_index": BoolOption(
+		comment="Hide wordhead from word definition in the tap-to-check interface.",
+	),
+	"spellcheck": BoolOption(
+		comment="The spell attribute enables wildcard search and spell "
+		"correction during word lookup.(May be it just enable the kindlegen's"
+		"spellcheck.)",
+	),
+	"exact": BoolOption(
+		comment="Exact-match Parameter.I guess it only works for inflections, "
+		"but have not yet give it a try.",
+	),
+	"newline_before_defi": BoolOption(
```

Comment:
We don't need an option for this. If definition starts with a newline, remove the newline outside template.

Reply:
Remove the newline outside template is easy for me.
Use a modified version of .mobi output plugin or some other scripts is also easy for me。
But how about **potential** users with the same problem? 
Can we assume they are all power-users?
I guess the answer is the same for why this project have so much variants  of GUIs while the CLI interface is "sufficient" and more powerful.


Simply is the best, but tools are made for people to simplify their tasks. 
So we must make a trade-off between them.

Anyway , it is you who make the decide.
I just ask you to think it again.
No hard feelings.

Reply:
> I meant remove it in the code.
?
I know what you mean.

If you just want to emphasize and repeat what you had said.
Then do what you want to do.
I SAID you have the right, and I have checked  `Allow edits by maintainers`.

I’m sorry if I misunderstood you reply. You know my English is not so good.